src/resource/solidDataset.ts

```diff
@@ -705,7 +705,9 @@ export async function saveSolidDatasetInContainer(
  * will do the job just fine, and does not require the parent Container to exist in advance.
  *
  * @param containerUrl URL of the Container in which the empty Container is to be created.
- * @param options Optional parameter `options.fetch`: An alternative `fetch` function to make the HTTP request, compatible with the browser-native [fetch API](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/fetch#parameters).
+ * @param options Optional parameter `options.fetch`: An alternative `fetch` function to make the HTTP request, compatible with the browser-native [fetch API](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/fetch#parameters).`options.slugSuggestion` accepts a string for your new Container's name.
```

Comment:
please rewrap this line, as it's "too long" and will likely anger the gods of eslint and prettier.

Reply:
Arguably you could just say `Optional parameter that expects the same values as